app.js

```diff
@@ -1,93 +1,109 @@
     alert('Nope, it was 6 public schools. I was a military brat and moved quite a lot growing up.');
     console.log(interestingFact);
-}
+  }
+} functionName();
 
-var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
-if (careerGoals.toLowerCase() === 'yes') {
+function careerGoals() {
+  var careerGoals = prompt('Do you think my ideal job is that as a Cartographer?');
+  if (careerGoals.toLowerCase() === 'yes') {
     alert('You must have a crystal ball! Yep, being a Cartographer is in fact my career goal');
     console.log(careerGoals);
-} else {
-    alert('Well almost... my career goal is in fact to be a Cartographer.')
+  } else {
+    alert('Well almost... my career goal is in fact to be a Cartographer.');
     console.log(careerGoals);
-}
+  }
+} careerGoals();
 
-var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
-if (focus.toLowerCase() === 'yes') {
+function focus(){
+  var focus = prompt('Is Python the computer language you think I\'m focusing my studies?');
+  if (focus.toLowerCase() === 'yes') {
     alert('Impressive, you are on the money!');
     console.log(focus);
-} else {
+  } else {
     alert('Actually, I\'m focusing my study on Python, as it is apparently the current standard in the GIS industry');
     console.log(focus);
-}
+  }
+} focus();
 
-var failedAttempts = [];
-var computerChoice = Math.floor((Math.random() * 10) + 1);
-var guessNumber = prompt('Guess a number between 1 and 10.');
-for (var i = 1; i < 4; i++) {
-  if (guessNumber === null) {
+//
+function functionName() {
+  var failedAttempts = [];
+  var computerChoice = Math.floor((Math.random() * 10) + 1);
+  var guessNumber = prompt('Guess a number between 1 and 10.');
+  for (var i = 1; i < 4; i++) {
+    if (guessNumber === null) {
       alert('You did not guess a number.');
       console.log(guessNumber);
       var guessNumber = prompt('Please guess a number.');
       alert(failedAttempts + ' was guessed already');
       continue;
-} if (guessNumber === computerChoice) {
+    } if (guessNumber === computerChoice) {
       alert('You guessed it!');
       console.log(guessNumber);
       alert(failedAttempts + ' was guessed already');
       break;
-} else if (guessNumber < computerChoice) {
+    } else if (guessNumber < computerChoice) {
       alert('Try Again!');
       failedAttempts.push(guessNumber);
       alert(failedAttempts + ' was guessed already');
       var guessNumber = prompt('Please guess another number between ' + guessNumber + ' and 10');
       continue;
-} else if (guessNumber > computerChoice) {
+    } else if (guessNumber > computerChoice) {
       alert('Try again! Pick another number between 1 and ' + guessNumber);
       failedAttempts.push(guessNumber);
       alert(failedAttempts + ' was guessed already');
       var guessNumber = prompt('Please pick another number between 1 and ' + guessNumber);
       continue;
     }
   }
+} functionName();
```

Comment:
your function declarations should end with a semicolon, and your function call should be on its own line